apps/src/music/views/PatternAiPanel.tsx

```diff
@@ -551,7 +551,7 @@ const PatternAiPanel: React.FunctionComponent<PatternAiPanelProps> = ({
   const aiBotImage = getAiBotImage();
 
   return (
-    <div className={styles.patternPanel}>
+    <div className={styles.patternPanel} dir="ltr">
```

Comment:
Maybe a quick comment to explain why we force LTR?

Reply:
Will add to a couple places in a follow-up!